region/maridia/outer.json

```diff
@@ -845,6 +845,25 @@
           "nodeSubType": "passage",
           "nodeAddress": "0x001a3cc",
           "doorEnvironments": [{"physics": "water"}],
+          "leaveWithGModeSetup": [
+            {
+              "strats": [
+                {
+                  "name": "Get Hit By Sciser",
+                  "notable": false,
+                  "requires": [
+                    "h_EverestMorphTunnelExpanded",
+                    "Morph",
+                    {"ammo": { "type": "Super", "count": 1 }}
+                  ],
+                  "note": [
+                    "Get to the item while avoiding the crab, note that it doesn't go all the way through the tunnel.",
```

Comment:
I don't think you can avoid a crab hit without some items but I'm not testing for it.  
You can pause abuse through but then it changes your reserve tank numbers and thats annoying.
Its mostly a problem because crabs are 1 hit kill when suitless.

Reply:
Yeah, I tried it with nothing. it's not too bad